src/test/java/games/strategy/engine/data/EngineDataEqualityComparators.java

```diff
@@ -0,0 +1,45 @@
+package games.strategy.engine.data;
+
+import games.strategy.test.EqualityComparator;
+
+/**
+ * A collection of equality comparators for engine data types.
+ */
+public final class EngineDataEqualityComparators {
```

Comment:
This class contains implementations of `EqualityComparator` for types under `g.s.engine.data`.  The implementations are identical to what existed prior to this PR.